.github/CODEOWNERS

```diff
@@ -711,6 +711,12 @@
 # ServiceLabel: %Network - Network Virtual Appliance
```

Comment:
We should bump this down to after the `Network - Mobile` entries to preserve ordering.